pkgcfg/package-configuration/mariadb.json

```diff
@@ -0,0 +1,17 @@
+{
+  "name": "mariadb",
+  "version": "0.0.1",
+  "env": {
+    "MYSQL_BASEDIR": "{{ .DevboxProfileDefault }}",
+    "MYSQL_HOME": "{{ .DevboxRoot }}/conf/mysql",
+    "MYSQL_DATADIR": "{{ .DevboxRoot }}/conf/mysql/data",
+    "MYSQL_UNIX_PORT": "{{ .DevboxRoot }}/conf/mysql/mysql.sock",
+    "MYSQL_PID_FILE": "{{ .DevboxRoot }}/conf/mysql/mysql.pid"
+  },
+  "create_files": {
+    ".devbox/conf/mariadb/bin/mysql": "/mariadb/mysql",
```

Comment:
should this be `"{{ .DevboxRoot }}/conf/mariadb/bin/mysql": ... ` ?

Reply:
The key is a relative path so it's not needed. I think allowing relative paths makes writing the configs a bit nicer